README.md

```diff
@@ -2,17 +2,25 @@
 
 ## About
 
-This [Github repository](https://github.com/stencila/policies) is where we keep Stencila's policies, code of conduct, guidelines etc. Everything is available under a Creative Commons Attribution-ShareAlike [license](https://github.com/stencila/policies/blob/master/LICENSE.md). Just like our code, we think these policies and guidelines should be open source too!
+This [Github repository](https://github.com/stencila/policies) is where we keep Stencila's policies, code of conduct,
+guidelines etc. Everything is available under a Creative Commons Attribution-ShareAlike
+[license](https://github.com/stencila/policies/blob/master/LICENSE.md). Just like our code, we think these policies and
+guidelines should be open source too!
 
-- [Code of Conduct](https://policies.stenci.la/conduct)
-- [Privacy Policy](https://policies.stenci.la/privacy)
+- [Code of Conduct](./conduct.html)
+- [Privacy Policy](./privacy.html)
+- [Terms of Service](./terms.html)
```

Comment:
These relative links won't work when clicked on from https://github.com/stencila/policies#readme. The only place those links are needed are in the README and in https://policies.stenci.la/index.html, so I don't think they need to be relative.